docs/learn/digital-assets/smart-contract-developers/getting-started.md

```diff
@@ -98,6 +89,9 @@ By default, the deployment will be to your local network and default Solidity ve
 - add the parameters of the `networks` and their related private keys
 - define the `etherscan` APIs to verify contracts on the networks
 
+<details>
+<summary>Final Hardhat config file from the [Playground repo](https://github.com/lukso-network/lukso-playground)</summary>
```

Comment:
Nice 👌🏻 Let's remove mainnet as well from the config file code as well. Since the guide focuses on testnet, mainnet configs are irrelevant